apps/admin/backend/src/app.ts

```diff
@@ -639,27 +638,23 @@ function buildApi({
       adjudicateCvrContest(input, store, logger);
     },
 
-    getVoteAdjudications(
-      input: {
-        contestId?: ContestId;
-        cvrId?: Id;
-      } = {}
-    ): VoteAdjudication[] {
+    getVoteAdjudications(input: {
+      contestId: ContestId;
+      cvrId: Id;
+    }): VoteAdjudication[] {
       return store.getVoteAdjudications({
-        ...input,
         electionId: loadCurrentElectionIdOrThrow(workspace),
+        contestId: input.contestId,
+        cvrId: input.cvrId,
       });
     },
 
-    getWriteInAdjudicationQueueMetadata(
-      input: {
-        contestId?: ContestId;
-        status?: WriteInAdjudicationStatus;
-      } = {}
-    ): WriteInAdjudicationQueueMetadata[] {
+    getWriteInAdjudicationQueueMetadata(input: {
```

Comment:
Note that this function will be deprecated in favor of `getWriteInAdjudicationCvrQueueMetadata`, as it is used for the previous write-in adjudication flow

Reply:
Feel free to take over this PR and do what you want with it, including just incorporating the relevant changes to your branch and closing this.

Reply:
Okay thanks, I'll incorporate the changes in a commit later today. I appreciate your help with this